PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -158,7 +158,7 @@ struct HfCandidateSelectorD0 {
       return false;
     }
     // candidate DCA
-    // if (candidate.chi2PCA() > cuts[pTBin][1]) return false;
+    if (candidate.impactParameterXY() > cuts[pTBin][1]) return false;
```

Comment:
@Libeiyu I think that you have to change it into 
```suggestion
    if (candidate.impactParameterXY() > cuts->get(pTBin, "DCA") return false;
```

Reply:
ahh, I see. I just mainly focus on the table, I should have a look at the log file
`  165 |     if (candidate.impactParameterXY() > cuts[pTBin][1]) return false;
      | `

Reply:
@mfaggin Thanks a lot!

Reply:
> @mfaggin Thanks a lot!

you are welcome :) I approve it, now it should compile and go (in case you can give a try locally it would be even better)

Reply:
Sure! This time it's just to save a bit of local updating and compiling time.